src/main/java/org/apache/sling/feature/cpconverter/index/IndexDefinitionsJsonWriter.java

```diff
@@ -145,7 +147,7 @@ private void write(JsonGenerator json, DocViewNode2 index, String parentPath) th
             json.writeStartObject(JcrConstants.JCR_CONTENT);
             String blobAsString = new String(binary.get(), StandardCharsets.UTF_8);
             write(json, JcrConstants.JCR_PRIMARYTYPE, Collections.singletonList(JcrConstants.NT_RESOURCE),  s -> Json.createValue("nam:" + s ));
-            write(json, JcrConstants.JCR_MIMETYPE,Collections.singletonList(Files.probeContentType(Paths.get(nodePath))), Json::createValue );
+            write(json, JcrConstants.JCR_MIMETYPE,Collections.singletonList(Files.probeContentType(Paths.get(FilenameUtils.getName(nodePath)))), Json::createValue );
```

Comment:
For JCR Paths I would rather use https://github.com/apache/jackrabbit/blob/trunk/jackrabbit-jcr-commons/src/main/java/org/apache/jackrabbit/util/Text.java.

Reply:
Maybe also the variable naming is just confusing because FS path name != repository path name. Not sure what node path is supposed to contain here.

Reply:
i've seend the nodePath is constructed a few lines above, using directly nodeName instead

Reply:
still nodeName != fileName due to the escaping rules of FileVault.

Reply:
i've applied PlatformNameFormat.getPlatformName